### DISCUSSION TITLE
FzfLuaNormal not working as I expected

### DISCUSSION THREAD

**Discussion by towry** [ibhagwan/fzf-lua#1160] [Q&A] [answered]
![image](https://github.com/ibhagwan/fzf-lua/assets/8279858/98d2ff9c-8df7-4e62-a6dd-4b41f33e5bad)

Can't make the upper main window's border and background matching to FloatBorder, seems no effect at all.

**Comment by ibhagwan** [comment]
`FzfLuaNormal` is just for the inner part of the window, for the border use `FzfLuaBorder`.

**Comment by ibhagwan** [comment]
Works perfectly as expected for me:
```lua
:FzfLua files hls.normal=IncSearch hls.border=Cursor
```

![IMG_2061](https://github.com/ibhagwan/fzf-lua/assets/59988195/5899f787-b5f6-4346-b729-fb21264e376a)

**Reply by towry** [reply]
![image](https://github.com/ibhagwan/fzf-lua/assets/8279858/f4371cc1-4ac1-4c79-b4dc-7e566ad2a123)

Seems doesn't work for me

**Reply by towry** [reply]
![image](https://github.com/ibhagwan/fzf-lua/assets/8279858/9d514489-9398-4138-8423-f22232c4db8d)

If i removed the fzf_colors option, this is how it looks.

**Reply by towry** [reply]
![image](https://github.com/ibhagwan/fzf-lua/assets/8279858/db40ffac-1525-430f-a626-dcccd8f7184c)

![image](https://github.com/ibhagwan/fzf-lua/assets/8279858/5d4aafea-6074-4513-82f5-c379396bd2a9)

**Comment by ibhagwan** [comment]
> If i removed the fzf_colors option, this is how it looks.

`fzf_colors` tells fzf to use colors directly for specific elements, it will override `FzfLuaNormal` in the elements you specified (`fg`, `bg`, etc).

I can’t tell what `NormalFloat` highlight group is so I don’t know if this works or not. Perhaps you got some other overrides in our config?

- Can you try with `mini.sh`?
- What happens if you run `:FzfLua files hls.normal=IncSearch hls.border=IncSearch` (I’m using `IncSearch` because it usually has very distinct background)?

**Reply by towry** [reply]
![image](https://github.com/ibhagwan/fzf-lua/assets/8279858/3ceded7b-c30f-46e5-896d-f96380cc0339)


I am using mini.sh.

**Reply by towry** [reply]
![image](https://github.com/ibhagwan/fzf-lua/assets/8279858/68553f5f-a358-49c6-933e-9c76de2bb617)

tested in kitty and wezterm, same

**Reply by towry** [reply]
NVIM v0.10.0-dev-build-20231109-1627-gefaf37a2b

fzf-lua: 2674675

fzf: 0.46.1 (brew)

**Reply by ibhagwan** [reply, marked as answer]
> NVIM v0.10.0-dev-build-20231109-1627-gefaf37a2b

Works for me on 0.9.5 but not on latest nightly, perhaps an issue with the nightly, I’m checking why this isn’t working as expected, ~~might be a recently added change or an upstream bug~~ fixed in #1161.
